src/zks_provider/mod.rs

```diff
@@ -845,10 +845,6 @@ impl<P: JsonRpcClient> ZKSProvider for Provider<P> {
         // Note: We couldn't implement ZKSWalletError::LexerError because ethers-rs's LexerError is not exposed.
         let function = if contract_address == zks_utils::ECADD_PRECOMPILE_ADDRESS {
             zks_utils::ec_add_function()
-        } else if contract_address == zks_utils::ECMUL_PRECOMPILE_ADDRESS {
-            zks_utils::ec_mul_function()
-        } else if contract_address == zks_utils::MODEXP_PRECOMPILE_ADDRESS {
-            zks_utils::mod_exp_function()
```

Comment:
I don't remember removing this. Was this an automatic merge accident?

Reply:
Yes, probably. Those lines exist in `main`